vue.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
             repo: 'MyVergies'
           }
         ],
+        asar: false,
```

Comment:
isn't asar necessary for quick auto updates?

Reply:
Not sure, will have to test. Also we can probably only unpack the needed stuff from the asar. In any case running executables from the asar isn't possible.

There is an `asarUnpack` option for E-builder.
Will look at it today.

Reply:
I meant I will look at it today if you already didn't 😄

Reply:
I didn't, and I also don't have time today.

Reply:
It's using the asar mode again.
We just unpack the bins now.
Had to copy some of the Granax code and make our own implementation.
I don't feel this is a huge problem.